frontend/app/mocks/node.ts

```diff
@@ -168,6 +168,63 @@ const handlers = [
       nameFr: preferredLanguageEntity.nameFr,
     });
   }),
+
+  /**
+   * Handler for POST requests to WSAddress parse service
+   */
+  http.post('https://api.example.com/address/parse', ({ params }) => {
+    return HttpResponse.json({  
+      responseType: "CA",
+      addressLine: "23 CORONATION ST",
+      city: "ST JOHNS",
+      province: "NL",
+      postalCode: "A1C5B9",
+      streetNumberSuffix: "streetNumberSuffix",
+      streetDirection: "streetDirection",
+      unitType: "unitType",
+      unitNumber: "000",
+      serviceAreaName: "serviceAreaName",
+      serviceAreaType: "serviceAreaType",
+      serviceAreaQualifier: "",
+      cityLong: "cityLong",
+      cityShort: "cityShort",
+      deliveryInformation: "deliveryInformation",
+      extraInformation: "extraInformation",
+      statusCode: "Valid",
+      canadaPostInformation: [],
+      message: "message",
+      addressType: "Urban",
+      streetNumber: "23",
+      streetName: "CORONATION",
+      streetType: "ST",
+      serviceType: "Unknown",
+      serviceNumber: "000",
+      country: "CAN",
+      warnings: '',
+      functionalMessages: [
+        {action: 'action',
+        message: 'message'},
+      ],
```

Comment:
**Nitpick:** there are a few places in this PR where it is evident that you are not auto-formatting your code.

I think this is perfectly fine (I personally hate autoformatters), but we've decided to use autoformatting in this project, which means that any future changes to this file (and others in this PR) will likely be cluttered with unrelated formatting diffs since future devs will likely be formatting their code on save.

My advice would be to just bite the bullet and autoformat your code (and set up a save action to do this in your IDE). If there are places in the code where you think the autoformatter is making the code less-readable, you can always add a `prettier-ignore` to tell it to back off. (see [`raoidc-utils.server.ts`](https://github.com/DTS-STN/canadian-dental-care-plan/blob/main/frontend/app/utils/raoidc-utils.server.ts#L114))

Reply:
Installed the prettier extension for VSCode and set it as the default formatter and to format on save, it appears to be auto formatting the file when I insert a extra space in. I also ran the npm lint and format command which just displayed warnings for all files but nothing specific for this PR.